docs/rules/no-use-before-define.md

```diff
@@ -82,7 +82,7 @@ function g() {
   Default is `true`.
 
 This rule accepts `"nofunc"` string as an option.
-`"nofunc"` is the same as `{ "functions": false, "classes": true }`.
+`"nofunc"` is the same as `{ "functions": false }`.
```

Comment:
Why this change? It looks like this is still the behavior (see https://github.com/eslint/eslint/blob/master/lib/rules/no-use-before-define.js#L26).  If anything, I can it should be

```suggestion
`"nofunc"` is the same as `{ "functions": false, "classes": true, "variables": true }`.
```

Reply:
ohh yes, this are loaded as default ! thanks for pointing 👍